### PR TITLE
Use .net 5 in timeseries domain

### PR DESCRIPTION
**Pull request by HenrikSommer**
## Description

This PR upgrades .net code in the Timeseries domain to .NET 5

MessageReceiver Startup class is replaced with Program class for a .NET 5 like implementation.
Our pipeline is updated to use .NET 5 isolated mode.
All temporary shared code is upgraded to .NET 5

### CODE REVIEW

**Comment by lasrinnil** [comment]
I think we need to update the message-receiver-cd.yml to target .Net 5.0

**Comment by lasrinnil** [comment]
I have pushed an update to the PR including the upgrade to .Net 5 on the ci/cd workflow for the message receiver.

File: source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.MessageReceiver/GreenEnergyHub.TimeSeries.MessageReceiver.csproj [hunk 1]
```diff
@@ -15,14 +15,17 @@ limitations under the License.
 -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
+    <LangVersion>9</LangVersion>
```
Comment (BjarkeMeier) [inline source/GreenEnergyHub.TimeSeries/source/GreenEnergyHub.TimeSeries.MessageReceiver/GreenEnergyHub.TimeSeries.MessageReceiver.csproj:19]: Ooh, nice :-)

**Review by BjarkeMeier — APPROVED** [review]
I assume that this is tested.
Does this impact (fix) our application map in app.insights?

**Review by Mech0z — APPROVED** [review]

**Pull request merged**